exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +8,14 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if(grade<50){
+    return "failed";
```

Comment:
```suggestion
    return "failed";
    } else if(grade >= 50) 
   {
    return "passed";
   }
```

Reply:
Thank you for reminding Jaspreet, I changed something at the function. It is better now.